### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by rhagarty**
- update readme format
- uses shared pngs for Watson Studio setup
- updated notebook to hide WML credentials
- added completed notebook as example

### CODE REVIEW

**Comment by rhagarty** [comment]
Please allow @SSaishruthi a chance to review

File: README.md [hunk 1]
```diff
@@ -1,96 +1,200 @@
...
+
+# Troubleshooting
+
+* Notebook error:
+
+  ![library-error](doc/source/images/library-error.png)
+  
+  This will occur if when you run the notebook multiple times. The custom library `NAME` found in the structure below must be unique for each run. Change the value and run the cell again.
+
+  ```bash
```
Comment (stevemar) [inline README.md]: use python instead here?

Comment (rhagarty) [reply README.md]: will try that. Also noticed that the sentence needs some fixin

**Review by SSaishruthi — APPROVED** [review]

**Review by stevemar — APPROVED** [review]
LGTM

**Comment by stevemar** [comment]
thanks @rhagarty !

**Pull request merged**